### PR TITLE
Grouping label's expand button with grouping id

### PR DESCRIPTION
**Pull request by m-masataka**
In alerts screen, Expand/Collapse button cooperated with grouping labels.
So when several groups have the same grouping labels, this button will expand all of them. Related to #1875 .
To fix this issue, I implemented expand button with grouping id. (this id is only used UI)

### CODE REVIEW

**Comment by m-masataka** [comment]
alertmanager

File: ui/app/src/Views/AlertList/Views.elm [hunk 1]
```diff
@@ -98,23 +98,23 @@ defaultAlertGroups activeId activeGroups expandAll groups =
...
         _ ->
             div [ class "pl-5" ]
-                (List.map
-                    (\{ labels, alerts } ->
-                        alertGroup activeId activeGroups (Dict.toList labels) alerts expandAll
-                    )
-                    groups
+                (List.indexedMap Tuple.pair groups
+                    |> List.map
+                        (\( index, group ) ->
```
Comment (w0rm) [inline ui/app/src/Views/AlertList/Views.elm]: There is no need to first do `indexedMap` to build tuples and then map over tuples. You can call `List.indexedMap` with `\index group ->`.

**Comment by w0rm** [comment]
A small improvement, everything else looks good! I think that after this change this should also perform better, because the key type of a Set is simpler. 

Thank you!

**Comment by simonpasquier** [comment]
Thanks a lot @m-masataka! You need to rebase your PR after #2013 is merged.

@stuartnelson3 @mxinden once this gets merged, I suggest that we consider a new release of AlertManager as #1875 is quite annoying and has been reported in different occasions. I can sign up for preparing the new release.

File: ui/app/src/Views/AlertList/Views.elm (hunk 1 above)
Comment (m-masataka) [reply ui/app/src/Views/AlertList/Views.elm]: Thanks!

**Pull request merged**